### ISSUE TITLE
record locking is broken

### ISSUE THREAD

**Issue by stevepiercy** [knop-project/knop#131]
Record locking fails with error messages, depending on whether one is trying to edit:

```
Keyfield or keyvalue missing 7006
```

or delete:

```
Record locked by another user 7010
```

a record in the Knop 9 demo under Customer Advanced > Edit.

In addition, an anonymous user can delete any record with a record lock through the Customers Simple interface.

**Comment by stevepiercy** [comment]
I have no idea why this does not work, but I isolated it to an issue with `knop_seed` and `blowfish_decrypt` using an instance variable set to the value of `knop_seed`.

## To replicate
* Checkout commit 8baeda31a6e4c01147484bfda505b194588684d0
* Login and navigate to Customers Advanced, click an item, edit it, then try to save it.

## Potential fix
* Change the definition of `knop_seed` in `knop_utils.lasso` by checking out commit 

## Test code used in a failed attempt to replicate the issue
https://gist.github.com/stevepiercy/5b8dd3d67edf7aa61671395d3ea167ab

**Comment by stevepiercy** [comment]
Fixed in 6e1b0547985cc78ed69861391fedea0635ee1441
